ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js

```diff
@@ -95,12 +99,37 @@
                     }, parseInt(autoSaveInterval) * 1000);
                 }
             }
+            this._model.subscribe((action) => {
+                const { payload } = action;
+                const { changes, field } = payload;
+                const { items } = field;
+                if (changes && changes.length > 0) {
+                    changes.forEach((change) => {
+                        switch (change.propertyName) {
+                            case "activeChild":
+                                FormContainerV2.hamburgerMenuInstance.handleActiveItem(field);
```

Comment:
There can be mutiple form container in a page, this needs to be part of the form container instance and not the classs